src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNINpHandle.cs

```diff
@@ -38,15 +37,14 @@ internal sealed class SNINpHandle : SNIPhysicalHandle
         private int _bufferSize = TdsEnums.DEFAULT_LOGIN_PACKET_SIZE;
         private readonly Guid _connectionId = Guid.NewGuid();
 
-        public SNINpHandle(string serverName, string pipeName, long timerExpire, object callbackObject)
```

Comment:
The `callbackObject` parameter can be removed in the following locations:

- `SNITcpHandle` ctor
- `SNIProxy.CreateNpHandle()`
- `SNIProxy.CreateTcpHandle()`
- `SNIProxy.CreateConnectionHandle()`